projects/angular/components/ui-file-picker/src/shared/file-drop-zone/file-drop-zone.directive.ts

```diff
@@ -0,0 +1,131 @@
+import {
+    AfterViewInit,
+ Directive, ElementRef, EventEmitter, HostListener, Input, OnInit, Output,
+} from '@angular/core';
+import {
+ Subject, map, takeUntil,
+} from 'rxjs';
+import { FileReaderService } from '../file-reader.service';
+import { sortAndFilter } from '../file-drop-zone.utils';
+
+// The highlight class gets added to the host HTML element,
+// so to display properly the component containing it
+// should have that class defined in its styles.
+const DROP_ZONE_HIGHLIGHT_CLASS = 'file-drop-zone-highlight';
```

Comment:
I would rename it like this
```suggestion
const DROP_ZONE_HIGHLIGHT_CLASS = 'ui-file-drop-zone-highlight';
```